cmd/kubectl_token.go

```diff
@@ -443,9 +443,10 @@ func samlAuth(input *LoginInput, client *http.Client) (managementClient.Token, e
 
 	client.Timeout = 300 * time.Second
 
-	loginRequest := fmt.Sprintf("%s/dashboard/auth/login?requestId=%s&publicKey=%s&responseType=%s",
+	loginRequest := fmt.Sprintf("%s/dashboard/auth/login?cli=true&requestId=%s&publicKey=%s&responseType=%s",
 		input.server, id, encodedKey, responseType)
 
+	customPrint(fmt.Sprintf("\nLogin Request Id: %s \n", id))
```

Comment:
Why do we add a space before the newline in these messages?

Reply:
will fix. no idea where it came from

Reply:
I am unable to push a fix on this branch/PR ?! (Ditto for the others)